sentry-android-core/src/main/java/io/sentry/android/core/internal/util/FirstDrawDoneListener.java

```diff
@@ -0,0 +1,95 @@
+package io.sentry.android.core.internal.util;
+
+import android.annotation.SuppressLint;
+import android.os.Build;
+import android.os.Handler;
+import android.os.Looper;
+import android.view.View;
+import android.view.ViewTreeObserver;
+import androidx.annotation.RequiresApi;
+import io.sentry.android.core.BuildInfoProvider;
+import java.util.concurrent.atomic.AtomicReference;
+import org.jetbrains.annotations.NotNull;
+
+/**
```

Comment:
`m:` I think we should put a comment here that we adapted it from Firebase. SOmething like "Adapted from <link> under Apache 2.0 license` or so